about.html

```diff
@@ -1,109 +0,0 @@
----
```

Comment:
Thought: Should we have a redirect for this page?

Reply:
df3af87